src/icons/flowTypes.js

```diff
@@ -0,0 +1,21 @@
+// @flow
+export type Icon = {
+    className?: string,
+    color?: string,
+    height?: number,
+    /** A text-only string describing the icon if it's not purely decorative for accessibility */
+    title?: string | React.Element<any>,
+    width?: number,
+};
+
+export type TwoTonedIcon = {
```

Comment:
Looks like this is only used in one place currently.

Reply:
Yes, for now it is. The hope is that we start adding more icons of this type going forward that have better customization options via CSS.